test/jdk/java/awt/print/bug8023392/bug8023392.java

```diff
@@ -46,9 +46,10 @@
 
 public class bug8023392 extends Applet {
     static final String[] instructions = {
+        "Please select variable radiobutton in applet.",
```

Comment:
why?  And the text reads oddly,  I think you mean
"Please select the Radio Button for applet size labeled "sariable"  in the test harness window"
but still, why ??

Reply:
Even now, in "fixed" mode the "Print" button is not visible and we need to select "Variable" to see the button. I just made it more official by changing the instruction.
BTW, I dont see the general comment about the label...I will work on the getting the inset than adding spaces in label.